### PR TITLE
[Code] Updated xaml formatting to current convention

### PR DESCRIPTION
**Pull request by Eideren**
See [Kryptos-FR's post](https://github.com/xenko3d/xenko/pull/161#issuecomment-416559222) for context.

### CODE REVIEW

**Comment by Eideren** [comment]
Another push is coming, a couple of other files weren't found through my tool.

**Comment by Eideren** [comment]
Done. I kept the rest of the changes down to spacing only, automated formatting often failed to produce decent results.

File: sources/editor/Xenko.PrivacyPolicy/PrivacyPolicyWindow.xaml [hunk 1]
```diff
@@ -9,25 +9,26 @@
...
-        <Run Text="{x:Static crashReport:CrashReportForm.PrivacyPolicyUrl}"/>
+      <Hyperlink TextDecorations="Underline" CommandParameter="{x:Static crashReport:CrashReportForm.PrivacyPolicyUrl}"
+                 Command="{x:Static xk:UtilityCommands.OpenHyperlinkCommand}">
+        <Run Text="{x:Static crashReport:CrashReportForm.PrivacyPolicyUrl}" />
       </Hyperlink>
     </TextBlock>
     <CheckBox IsChecked="{Binding PrivacyPolicyAccepted, RelativeSource={RelativeSource AncestorType=Window}}"
               IsEnabled="{Binding CanAccept, RelativeSource={RelativeSource AncestorType=Window}, Mode=OneWay}"
-              Content="I agree to the Privacy Policy" DockPanel.Dock="Bottom" Margin="20,0"/>
+              Content="I agree to the Privacy Policy" DockPanel.Dock="Bottom" Margin="20,0" />
```
Comment (xen2) [inline sources/editor/Xenko.PrivacyPolicy/PrivacyPolicyWindow.xaml]: @Kryptos-FR do we want those space changes too?

Comment (Kryptos-FR) [reply sources/editor/Xenko.PrivacyPolicy/PrivacyPolicyWindow.xaml]: Doesn't seem that we are consistent on that regard so it doesn't matter much.

**Comment by xen2** [comment]
Thanks!

**Pull request merged**